msi/GitProduct.wxs

```diff
@@ -1,78 +1,283 @@
-<?xml version='1.0'?>
```

Comment:
@dscho Could you please help me and tell me how I might "normalize" the line endings in this file so that we can get a good and proper diff? Stupid me didn't have my global git configuration set properly.